compile.py

```diff
@@ -15,23 +15,31 @@
 REPO = config.REPO
 github = Github(config.TOKEN)
 
+SEVERITY_LABELS = ['Severity: Critical Risk', 'Severity: High Risk', 'Severity: Medium Risk', 'Severity: Low Risk', 'Severity: Gas Optimization', 'Severity: Informational']
+
 issue_dict = {}
 
 # TODO catch get_repo() 404 errors and produce a gentle suggestion on what's wrong.
 
-for issue in github.get_repo(REPO).get_issues():
+github_issues = github.get_repo(REPO).get_issues()
+for issue in github_issues:
     # TODO what happens when there are more than one issue labels. In that
     # case, need to put the issue into the label with highest severity.
-    if len(issue.labels) != 0:
-        label = issue.labels[0].name
-        if label not in issue_dict:
-            issue_dict[label] = []
-        issue_dict[label].append(f"### {issue.title} \n{issue.body}\n")
+    if len(issue.labels) != 0 and issue.state == 'open':
+        # get issue labels as an array
+        issue_labels = list(map(lambda x: x.name, issue.labels))
+
+        # filter issue labels for only severity labels
+        severity_labels_in_issue = list(filter(lambda x: x in issue_labels, SEVERITY_LABELS))
```

Comment:
```suggestion
        # filter issue labels for only severity labels
        severity_labels_in_issue = [label for label in issue_labels if label in SEVERITY_LABELS]
```
A nice example of list comprehension!